src/tribler-common/tribler_common/sentry_reporter/sentry_tools.py

```diff
@@ -112,21 +112,23 @@ def distinct_by(list_of_dict, key):
     return result
 
 
-def skip_dev_version(version):
-    """
-    For the release version let's ignore all "developers" versions
-    to keep the meaning of the `latest` keyword:
-    See Also:https://docs.sentry.io/product/sentry-basics/search/
-    Args:
-        version: version string
-
-    Returns: version if it is not a dev version, and Null otherwise
-
-    """
+def format_version(version):
     if not version:
         return version
 
+    # For the release version let's ignore all "developers" versions
+    # to keep the meaning of the `latest` keyword:
+    # See Also:https://docs.sentry.io/product/sentry-basics/search/
     if 'GIT' in version:
         return None
 
-    return version
+    parts = version.split('-', maxsplit=2)
+    if len(parts) < 2:
+        return version
+    
+    # in case of a release candidate, leave "<version>-RC<number>"
+    if parts[1].lower().startswith('rc'):
```

Comment:
There could be other suffixes besides RC eg. exp for experimental so it might be better to use regular expression to check for the version here allowing any alphanumeric suffix

Reply:
fixed